src/api/core/queries/baseQuery.ts

```diff
@@ -11,6 +11,7 @@ export enum QueryAction {
   insert = "insert",
   update = "update",
   delete = "delete",
+  completeUpdate = "completeUpdate",
```

Comment:
the request method needs to be `PUT` for attach/detach, do you have a suggestion for this name?

Reply:
Well, I definitely do not "completeUpdate". I would suggest:
action - well, it's quite repetitive, but why not?
effect
claim
force
impact
Choose what you like :)

Reply:
got rid of this enum